src/tests/test_target.py

```diff
@@ -17,6 +17,7 @@ def setup_method(self, method):
         self.tgt_opt = forcebalance.parser.tgt_opts_defaults.copy()
         self.ff = None  # Forcefield this target is fitting
         self.options.update({'root': os.path.join(os.getcwd(), 'files')})
+        self.check_grad_fd = True # Whether to check gradient vs. finite difference. Set to False for liquid targets.
```

Comment:
@j-wags This was one of the changes; comparing analytic grad. to finite diff. would automatically fail for liquid properties because of the stochastic / statistical nature.